dist/rome.js

```diff
@@ -4276,7 +4275,10 @@ function CustomEvent (type, params) {
     function calendar$1(time, formats) {
         // Support for single parameter, formats only overload to the calendar function
         if (arguments.length === 1) {
-            if (isMomentInput(arguments[0])) {
+            if (!arguments[0]) {
+                time = undefined;
+                formats = undefined;
+            } else if (isMomentInput(arguments[0])) {
```

Comment:
if building changed this as is then I'm sure it's ok... just a little confused how it would've picked up a change like this. there haven't been any additional changes to the repo besides the one you made, so I would've expected only your new code to be added in this file